python/pyarrow/tests/test_serialization.py

```diff
@@ -520,6 +537,107 @@ def deserializer(data):
     assert np.alltrue(new_x.view(np.ndarray) == np.zeros(3))
 
 
+@pytest.mark.skip
```

Comment:
What is the reason why this test is skipped?

Reply:
I left that in by mistake. Removed.